src/api/v4/filter/category.js

```diff
@@ -67,8 +69,8 @@ class Category extends SQLBase {
     return {
       in: '<% if (value) { %><%= column %> IN (<%= value.query || value %>)<% } else { %>true = false<% } %>',
       notIn: '<% if (value) { %><%= column %> NOT IN (<%= value.query || value %>)<% } %>',
-      eq: '<%= column %> = <%= value %>',
-      notEq: '<%= column %> != <%= value %>',
+      eq: '<%= column %> = <%= value.query ? "(" + value.query + ")" : value %>',
```

Comment:
Why not `(<%= value.query || value %>)` as in `in`,`notIn`?

Reply:
I wouldn't like to include an unnecessary parenthesis in every query.
I can inject an `queryValueWithParenthesis` in the template and change it to `<%= queryValueWithParenthesis || value %>` if you think it's better :)

Reply:
Yep, I didn't think about it too much 😂